toolz/itertoolz.py

```diff
@@ -25,7 +24,7 @@ def remove(predicate, seq):
     >>> list(remove(iseven, [1, 2, 3, 4]))
     [1, 3]
     """
-    return filterfalse(predicate, seq)
+    return itertools.filterfalse(predicate, seq)
```

Comment:
I would ever-so-slightly prefer to import `filterfalse` and `zip_longest` from `itertools` so they are available in the globals.

Reply:
I have no problem with that.